mysqld_exporter.go

```diff
@@ -262,6 +264,13 @@ const (
 		  FROM information_schema.schemata
 		  WHERE SCHEMA_NAME NOT IN ('mysql', 'performance_schema', 'information_schema')
 		`
+	queryResponseCheckQuery = `SELECT @@query_response_time_stats`
+	queryResponseTimeQuery = `
+		SELECT
+		    TIME, COUNT, TOTAL
+		  FROM information_schema.query_response_time
```

Comment:
Total would have to be exposed separately, usually we'd just expose the sum of the total as part of the histogram.